spec/spec_helper_local.rb

```diff
@@ -60,117 +60,226 @@
 shared_context 'Fedora 17' do
   let :facts do
     {
+      # Legacy
       id: 'root',
```

Comment:
Where is this even used? Can't this fact (and the modern version of it) simply be dropped instead?

Reply:
Will likely remove at the end, have left it in place as I complete the rest of the code changes

Reply:
Legacy facts have now been removed.